packages/fxa-auth-server/lib/oauth/jwt_access_token.js

```diff
@@ -27,6 +29,16 @@ exports.create = async function generateJWTAccessToken(accessToken, grant) {
     ? TOKEN_SERVER_URL
     : clientId;
 
+  // Ensure we don't mix PPIDs and 'profile' scope or sub-scopes. See #5826 and
+  // also the PPID docs: https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/oauth/pairwise-pseudonymous-identifiers.md#ensuring-3rd-parties-are-unable-to-access-user-information
```

Comment:
For the comment here to be accurate about "don't mix PPIDs and `profile` scope", we would need to forbid the FPN client from ever creating tokens with `profile` scope.